app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -119,22 +120,12 @@ $after-button-padding-left: govuk-spacing(5);
 // Top level navigation and search.
 .gem-c-layout-super-navigation-header__navigation-items,
 .gem-c-layout-super-navigation-header__search-items {
-  background: govuk-colour("light-grey");
   display: block;
   list-style: none;
-  margin: 0 (0 - govuk-spacing(3));
   padding: 0;
-
-  @include govuk-media-query($from: "tablet") {
-    margin: 0 (0 - govuk-spacing(6));
-  }
-
-  @include govuk-media-query($from: "desktop") {
-    background: none;
-    float: left;
-    padding: 0;
-    margin: 0;
-  }
+  background: none;
+  float: left;
```

Comment:
Could I check whether it's necessary to float the container here, just because I think it's good to avoid floating things if possible. It looks like the float is used for stopping the items overflowing the nav when the menu is closed but this could be avoided if we didn't set `inline-block` on .`gem-c-layout-super-navigation-header__navigation-items` but kept it as `block` - this could be okay since we only have now one item inside `gem-c-layout-super-navigation-header__navigation-items` so we don't need to use `inline-block` to align the items anymore, at least on mobile. Here's what I mean but I might be missing a reason why this won't work:

<img width="1174" alt="Screenshot 2022-11-24 at 18 11 30" src="https://user-images.githubusercontent.com/5007934/203846609-915e94e5-d1a8-485f-bdbc-8fd32bb27a58.png">